recipe/meta.yaml

```diff
@@ -100,33 +92,26 @@ outputs:
 
     {% set tests_to_skip = "_not_a_real_test" %}
 
-    # https://github.com/numpy/numpy/issues/3858 - could be related
-    {% set tests_to_skip = tests_to_skip + " or test_big_arrays" %}  # [s390x]
-    # https://github.com/numpy/numpy/issues/16046
-    {% set tests_to_skip = tests_to_skip + " or test_gcd_overflow" %}  # [s390x]
-    # skip simd tests because vxe2 feature is not detected when running from prefect
-    {% set tests_to_skip = tests_to_skip + " or test_features" %}  # [s390x]
     # https://github.com/numpy/numpy/issues/27313 - can be removed with clang >=16 update
     {% set tests_to_skip = tests_to_skip + " or (test_unary_spurious_fpexception and (f-cos or f-sin))" %}  # [osx and arm64]
```

Comment:
Can you see if this can be removed by setting the compiler to clang 17? (now that it has been published)
Also this may remove the need for the patch you added.

Reply:
I gave that a shot on the previous version and it [does indeed ](https://github.com/AnacondaRecipes/numpy-feedstock/pull/111) look like it can be removed once we use clang 17.